package.json

```diff
@@ -27,7 +27,8 @@
     "prettier:check": "prettier --check .",
     "test": "yarn workspaces foreach --parallel --topological-dev --verbose run test",
     "publish": "tsx scripts/publish.ts",
-    "play": "tsx scripts/play.ts"
+    "play": "tsx scripts/play.ts",
+    "yalc:slice-machine": "tsx scripts/yalc-sm.ts"
```

Comment:
💡 Is it not more `yalc:editor-packages`?

Reply:
Sounds good, didn't really know what to name it. Changed ✅